lib/sidekiq/api.rb

```diff
@@ -52,17 +52,7 @@ def queues
     end
 
     def fetch_stats!
-      pipe1_res = Sidekiq.redis { |conn|
-        conn.pipelined do
-          conn.get("stat:processed")
-          conn.get("stat:failed")
-          conn.zcard("schedule")
-          conn.zcard("retry")
-          conn.zcard("dead")
-          conn.scard("processes")
-          conn.lrange("queue:default", -1, -1)
-        end
-      }
+      pipe1_res = QuickStats.new.fetch_stats!
```

Comment:
note, this isn't the same as the previous `pipe1_res` we threw it into a hash to and did a few basic conversions for convenience.